### PR TITLE
CVP-4320: Increase requests memory for v4.15+

### PR DESCRIPTION
**Pull request by asergienk**

### CODE REVIEW

**Comment by asergienk** [comment]
/pj-rehearse periodic-ci-redhat-openshift-ecosystem-cvp-ocp-4.18-cvp-common-claim

**Comment by asergienk** [comment]
/pj-rehearse periodic-ci-redhat-openshift-ecosystem-cvp-ocp-4.17-cvp-fips-common-claim

**Comment by asergienk** [comment]
/pj-rehearse periodic-ci-redhat-openshift-ecosystem-cvp-ocp-4.16-cvp-common-claim

**Comment by asergienk** [comment]
/pj-rehearse periodic-ci-redhat-openshift-ecosystem-cvp-ocp-4.15-cvp-fips-common-claim

File: ci-operator/jobs/infra-periodics.yaml [hunk 1]
```diff
@@ -60,7 +60,7 @@ periodics:
           cpu: 500m
     serviceAccountName: upgrader
 - agent: kubernetes
-  cluster: app.ci
+  cluster: build10
```
Comment (nmars) [inline ci-operator/jobs/infra-periodics.yaml:63]: Were we advised to switch to `build10`?

Comment (asergienk) [reply ci-operator/jobs/infra-periodics.yaml:63]: there was a job `ci/prow/ordered-prow-config` failure that recommended to switch to build10. This PR will require DPTP team approval, let me ask them to take a look first

Comment (asergienk) [reply ci-operator/jobs/infra-periodics.yaml:63]: @nmars got in approval from Bear, please ltm when you get the chance

**Pull request merged**